roles/online_docs/templates/mkdocs/docs/apptainer.md

```diff
@@ -0,0 +1,595 @@
+
+* Prevent caching downloaded files from online repositories
+
+`$ apptainer pull --disable-cache centos.sif docker://centos:7.9`
+
+## Isolated runs
+
+**Be careful, by default some folders are automatically shared between host machine and container.** This depends on the options with which container was built and container runtime parameters. Most commonly, on host system user's current working directoy is automatically as `/home` folder inside the container. Badly written or malicious software from inside container, could change and delete current working directory or `/home/` directory on host system. Apptainer provides options to control this exposure, by controlling mount points. User can either limit with arguments `-c` or `-C`
+
+```
+    $ apptainer run -C mycontainer.sif
+```
+
+where
+
+```
+       -c, --contain[=false]        use minimal /dev and empty other directories (e.g. /tmp and $HOME) instead of sharing filesystems from your host
+       -C, --containall[=false]     contain not only file systems, but also PID, IPC, and environment
+```
+
+First option will share no folders, while the second one will (inside container) show only processes of the container.
+
+The `--no-mount` flag allows specific system mounts to be disabled, even if they are set in the `apptainer.conf` configuration file by the administrator.
+
+## Mounting host folders inside containers
+
+User can also manually control where the individual hosts folder will be mounted to, by using `--bind` (or short `-B`) argument. For example
+
+```
+    apptainer shell --bind /home/SOMEUSER/mydata:/mnt busybox.sif
+```
+
+this will expose host machine users `mydata` folder to `/mnt` folder inside container. User can instead also use `--mount` and `,ro` argument to expose a folder as a read-only inside container, f.e.
+
+```                                                                             
+    apptainer shell --mount type=bind,source=/home/SOMEUSER/mydata,destination=/mnt,ro busybox.sif
+```
+
+For more information, check Apptainer's website [about mount points](https://apptainer.org/docs/user/main/bind_paths_and_mounts.html).
+
+
+## Other considerations
+
+### Architecture considerations
+
+Software build for specific architecture, can be only executed on the systems of the same architecture. For example, an apptainer image build on Apple M1 (ARM) or PowerPC (RISC) architecture will not run on x86 systems and vice versa.
+
+The software compiled within the apptainer image can also have issues when running on same type of architecture, but older genration. For example, the software inside the image compiled on a modern x86 architecture CPU and with high level of optimization, can fail from running on older architecture CPU. This happens because compiled binary contains instructions of modern processor, that older ones cannot understand. Running software build on older architectures, should work on any new architecture of a same type.
```

Comment:
```suggestion
Software compiled within the _Apptainer_ image can also have issues when running on an older generation of the same type of architecture. This can happen when the compiled binary code contains instructions for a modern processor, that older ones cannot understand. The other way around, running software build for processors of a certain architecture should work on any newer generation processors of the same architecture.
```